LibMAD return fix from upstream (never added to SDL_mixer 1.2.13)

--- SDL_mixer-1.2.13.orig/music_mad.c
+++ SDL_mixer-1.2.13/music_mad.c
@@ -224,7 +224,7 @@
   if ((mp3_mad->status & MS_playing) == 0) {
 	/* We're not supposed to be playing, so send silence instead. */
 	memset(stream, 0, len);
-	return;
+	return 1;
   }
 
   out = stream;
